src/release/breaking-changes/index.md

```diff
@@ -46,6 +48,8 @@ release, and listed in alphabetical order:
 [Deprecated `RouteInformation.location`]: {{site.url}}/release/breaking-changes/route-information-uri
 [Customize tabs alignment using the new `TabBar.tabAlignment` property]: {{site.url}}/release/breaking-changes/tab-alignment
 [Updated EditableText scroll into view behavior]: {{site.url}}/release/breaking-changes/editable-text-scroll-into-view
+[Deprecate describeEnum]: {{site.url}}/release/breaking-changes/describe-enum
+[Make EnumProperty be type stricter]: {{site.url}}/release/breaking-changes/enum-property-type-stricter
```

Comment:
Wait! Wait a minute. I never saw this "Deprecate describeEnum" PR! I would never have approved this. Why can't these two breaking changes be merged into one breaking change? Oy, we have a policy where we never approve breaking change PRs that don't have a "landed in" field filled out. I must have been on vacation when this one happened.

Reply:
They can. Someone approved that (probably by mistake) and there were some adjustments to make, like this property that appeared later. I'll merge them.

Reply:
I don't want to incriminate anyone, but you merged it lol 
![Uploading image.png…]()